Dockerfile-mongo-seed

```diff
@@ -0,0 +1,13 @@
+FROM  mongo:3.2
+
+COPY dc-campaign-finance-mongodatabase.zip .
+
+# update packages and intall unzip
+RUN apt-get update
+RUN apt-get install unzip
+
+# unzip db .bson files to root dir
+RUN unzip dc-campaign-finance-mongodatabase.zip -d .
+
+# restore mongodb using docker compose name
+CMD mongorestore --host mongodb -d dc-campaign-finance --drop ./dc-campaign-finance-mongodatabase/dc-campaign-finance
```

Comment:
My preference would be to have the exported mongodb mounted as a volume so we don't have to rely on the zip being up to date in the repo.